drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -409,15 +550,22 @@ local function initialize_switch(driver, device)
     device:set_field(COMPONENT_TO_ENDPOINT_MAP_BUTTON, component_map, {persist = true})
   end
 
-  if #switch_eps > 0 then
+  if num_valve_server_eps > 0 then
```

Comment:
If you didn't already, we can remove this as well @nickolas-deboom

Reply:
Done in latest commit!